_i18n/es/resources/user-guides/monero-wallet-cli.md

```diff
@@ -1,147 +1,114 @@
 
-This guide will show how to perform various operations from the `monero-wallet-cli` UI. The guide assumes you are using the most recent version of Monero and have already created an account according to the other guides.
+Esta guía mostrará cómo realizar varias operaciones desde la interface de usuario `monero-wallet-cli`. Esta guía asume que estás versión más reciente de Monero y que ya has creado una cuenta conforme a las demás guías.
 
 
-## Checking your balance
+## Revisando tu balance
 
-Since the blockchain handling and the wallet are separate programs, many uses of `monero-wallet-cli`
-need to work with the daemon. This includes looking for incoming transactions to your address.
-Once you are running both `monero-wallet-cli` and `monerod`, enter `balance`.
+Ya que el manejo de la blockchain y del monedero son programas separados, varios usos de `monero-wallet-cli` necesitan trabajar con el daemon. Esto incluye buscar por transacciones de entrada a tu dirección. Una vez que estés corriendo `monero-wallet-cli` y `monerod`, escribe `balance`.
 
-Example:
+Ejemplo:
 
-This will pull blocks from the daemon the wallet did not yet see, and update your balance
-to match. This process will normally be done in the background every minute or so. To see the
-balance without refreshing:
+Esto sacará bloques del daemon que el monedero aún no ve, y actualizará tu balance para que coincidan. Este proceso será normalmente realizado en segundo plano más o menos cada minuto Para ver tu balance sin refrescar:
 
     balance
     Balance: 64.526198850000, unlocked balance: 44.526198850000, including unlocked dust: 0.006198850000
 
-In this example, `Balance` is your total balance. The `unlocked balance` is the amount currently available to spend. Newly received transactions require 10 confirmations on the blockchain before being unlocked. `unlocked dust` refers to very small amounts of unspent outputs that may have accumulated in your account.
+En este ejemplo, `Balance` es tu balance total. El `unlocked balance` es la cantidad actualmente disponible para gastar. Transacciones recientemente recibidas requieren 10 confirmaciones en la blockchain antes de ser desbloqueadas. `unlocked dust` se refiere a muy pequeñas cantidades de salidas sin gastar que se pueden haber acumulado en tu cuenta.
 
-## Sending monero
+## Enviando Monero
 
-You will need the standard address you want to send to (a long string starting with '4'), and
-possibly a payment ID, if the receiving party requires one. In that latter case, that party
-may instead give you an integrated address, which is both of these packed into a single address.
+Necesitarás la dirección estándar a la que quieres enviar (una larga cadena que comienza con '4'), y posiblemente un ID de pago, si el equipo receptor requiere uno. En ese último caso, el equipo te debería dar una dirección integrada, que es la combinación de ambas en una dirección individual.
```

Comment:
It can be applied to the first "party" part, but for the second it sounds a little weird, it may be better:

_En ese último caso, el receptor te debería dar una dirección integrada, ..._

Reply:
Excelent :)!